modules/lvms-adding-a-storage-class.adoc

```diff
@@ -0,0 +1,55 @@
+// This module is included in the following assemblies: 
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_content-type: PROCEDURE
+[id="adding-a-storage-class_{context}"]
+= Adding a storage class
+
+You can add a storage class to an `LVMCluster`. A storage class describes a class of storage in the `LVMCluster` and how the cluster dynamically provisions the persistent volumes (PVs) when the user specifies the storage class. A storage class can describe the type of storage such as solid state drives (SSDs), the quality-of-service level, the filesystem type, and other details. 
```

Comment:
In last line, can we update "solid state drives (SSDs)" to just "device-classes" as it is specific to lvms storage class?